naresh/src/main/java/com/lftechnology/phpjava/ems/entities/User.java

```diff
@@ -0,0 +1,61 @@
+package com.lftechnology.phpjava.ems.entities;
+
+/**
+ * User
+ *
+ * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+ * @since August, 08 2016
+ */
+public class User {
+    private final static String TABLE = "users";
+    protected int id;
+    protected String username;
+    protected String password;
+    protected boolean isTerminated;
+
+
+    public User() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1186)